ocaml/idl/datamodel.ml

```diff
@@ -174,6 +174,35 @@ module Session = struct
       ~in_product_since:rel_george ~in_oss_since:None ~allowed_roles:_R_POOL_OP
       ()
 
+  let get_all_records =
+    call ~name:"get_all_records" ~lifecycle:[] ~params:[] ~flags:[`Session]
+      ~result:
+        ( Map (Ref _session, Record _session)
```

Comment:
This is still a problem: you can't have a `get_all_records` function for sessions for security reasons.